.dockerignore

```diff
@@ -1,5 +1,4 @@
 *~
 **/.terraform/
-terraform/
```

Comment:
So removing it from dockerignore won't just copy whole dir into container, you need to use `COPY` in dockerfile to choose what you want to copy
It just adds terraform dir to docker daemon context. If dir is super large it can be problem, but terraform dir is not that large...

Reply:
In other images' build file, we do a full copy of the git repo.  This will cause their build to miss layer cache.

Reply:
Looking at it 2nd time, this is fine because most of us don't touch terraform config, and it will be transparent.